src/main/kotlin/com/sourcegraph/cody/history/state/HistoryState.kt

```diff
@@ -5,10 +5,18 @@ import com.intellij.util.xmlb.annotations.OptionTag
 
 class HistoryState : BaseState() {
 
-  @get:OptionTag(tag = "chats", nameAttribute = "") var chats: MutableList<ChatState> by list()
+  @get:OptionTag(tag = "accountHistories", nameAttribute = "")
+  var accountHistories: MutableList<AccountHistoryState> by list()
```

Comment:
Before we introduce more strange naming... Are we sure about `accountHistories`? IMHO the word "history" is redundant and misleading. And has been introduced probably in the context of Chat History panel.

Reply:
How about a separate PR to change the existing names? It may be worth doing that _before_ merging this PR.

Reply:
We cannot freely change that names, because they are used to seriallize the history.
When they are out and someone started to use it they is written in the stone.

Although, I think it should be possible to use some mapping, possibly with that `nameAttribute` property from above.

Reply:
I would vote for just name `AccountData`.
`AccountState` sounds more like info about account and in fact, we have class which is called `AccountState` already,

Reply:
Done